src/DynamoCore/Graph/Nodes/DummyNode.cs

```diff
@@ -370,6 +466,16 @@ internal string GetDescription()
         /// </summary>
         public string LegacyAssembly { get; private set; }
 
+        /// <summary>
+        /// Type name of the node.
+        /// </summary>
+        public string TypeName { get; private set; }
```

Comment:
any reason not to overload the LegacyNodeName - maybe note that this is property is only valid for ZT dummy nodes?